.editorconfig

```diff
@@ -9,7 +9,7 @@ root = true
 
 # Change these settings to your own preference
 indent_style = space
-indent_size = 4
+indent_size = 2
 
```

Comment:
Yeah, absolutely, indentation PR has to have only spaces changed and nothing else.

We had this with Meanjs and it was good to watch for a moment with minimum amount of PR's pending.